packages/backend/server/routes/stats.get.ts

```diff
@@ -0,0 +1,74 @@
+export default eventHandler(async (event) => {
+  try {
+    const binding = useBinding(event);
+
+    let cursor: string | undefined = undefined;
+    let totalObjects = 0;
+    let totalSize = 0; 
+    const orgs = new Set<string>();
+    const packages = new Set<string>();
+    const commits = new Set<string>(); 
+
+    const prefix = `${usePackagesBucket.base}:`;
+
+    do {
+      const result = await binding.list({ prefix, cursor });
+
+      const keys = result.objects.map((obj) => obj.key);
+      totalObjects += keys.length;
+
+      for (const obj of result.objects) {
+        const key = obj.key;
+        const size = obj.size || 0;
+        totalSize += size;
+
+        if (!key.startsWith(prefix)) {
+          continue;
+        }
+
+        const trimmedKey = key.slice(prefix.length);
+
+        const parts = trimmedKey.split(":");
+
+        if (parts.length >= 3) {
+          const org = parts[0];
+          const pkg = parts[1];

```

Comment:
no this is not package, pay attention to how we deal with keys, see other files.